### PR TITLE
Merge in CVE 2016-2785

### PR DESCRIPTION
**Pull request by KevinCorcoran**
There were no conflicts in the merge.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -1,8 +1,15 @@
+(defn deploy-info
```
Comment (camlow325) [inline project.clj]: @KevinCorcoran We should probably avoid pulling in any of the changes to this file except for the `:source-paths` and `:java-source-paths`.  We wouldn't want to have the project built and deployed to a non-Clojars repo.

Comment (KevinCorcoran) [inline project.clj]: good catch

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 1.5.5-CVE_2016_2785_1
+
+This is a security release.
+
+* [TK-343](https://tickets.puppetlabs.com/browse/TK-343) Support a new
+  option for handler registrations, `normalize-request-uri`, which can be
+  used to request that the URI path component is sanitized before the
+  handler is invoked for a request and that `.getRequestURI` calls made by
+  the handler return a path that has been percent-decoded.
+
```
Comment (KevinCorcoran) [inline CHANGELOG.md]: wow, I didn't even notice that you'd done this already!  jb++

Comment (camlow325) [inline CHANGELOG.md]: Cool, yeah, should just be able to swap out the version number on this note.

**Comment by camlow325** [comment]
Looks good to me other than fixing up what we're pulling in for the project.clj and the version string in the CHANGELOG.md.

File: project.clj (hunk 1 above)
Comment (KevinCorcoran) [inline project.clj]: fixed

**Comment by KevinCorcoran** [comment]
@camlow325 feedback addressed

**Comment by camlow325** [comment]
:+1: LGTM.  Can wait for Travis to complete its run before we merge.

**Comment by camlow325** [comment]
Travis passed, so merging...

**Pull request merged**